stdlib/string.pyi

```diff
@@ -64,6 +64,15 @@ class Formatter:
     ) -> LiteralString: ...
     @overload
     def vformat(self, format_string: str, args: Sequence[Any], kwargs: Mapping[str, Any]) -> str: ...
+    def _vformat(
+        self,
+        format_string: str,
+        args: Sequence[Any],
+        kwargs: Mapping[str, Any],
+        used_args: Sequence[int | str],
```

Comment:
```suggestion
        used_args: set[int | str],
```
It does `used_args.add`.

Reply:
I notice `self.check_unused_args(used_args, args, kwargs)` in method `vformat`, but :
```pyi
def check_unused_args(self, used_args: Sequence[int | str], args: Sequence[Any], kwargs: Mapping[str, Any]) -> None: ...
```
Maybe need add `@overload` for `check_unused_args` stub?

Reply:
I think that annotation is just wrong. It should also be `set[int | str]`.

Reply:
resolved.